simulator/func_sim/syscall/mars_calls.cpp

```diff
@@ -0,0 +1,81 @@
+/*
+ * mars_syscalls.cpp - implementation of MARS syscalls
+ * @author Vyacheslav Kompan kompan.vo@phystech.edu
+ * Copyright 2018 MIPT-MIPS
+ */
+
+#include "mars_calls.h"
+
+enum Regno : uint8 {
+    v0 = 2,
+    a0 = 4
```

Comment:
I think we may add access to registers by their name to avoid this workarounds

Reply:
Maybe `static const constexpr MIPSRegister v0 = MIPSRegister::from_cpu_index( 2);` does the trick.